antarest/study/web/study_data_blueprint.py

```diff
@@ -524,6 +525,41 @@ def update_inflow_structure(
         study = study_service.check_study_access(uuid, StudyPermissionType.WRITE, params)
         study_service.hydro_manager.update_inflow_structure(study, area_id, values)
 
+    @bp.get(
+        "/studies/{uuid}/{area_id}/load/series",
+        tags=[APITag.study_data],
+        summary="Get load series data",
+    )
+    def get_load_series(
+        uuid: str,
+        area_id: str,
+        current_user: JWTUser = Depends(auth.get_current_user),
+    ) -> Response:
+        """Return the load matrix in ARROW format."""
+        logger.info(
+            msg=f"Getting load series data for area {area_id} of study {uuid}",
+            extra={"user": current_user.id},
+        )
+        params = RequestParameters(user=current_user)
+        study = study_service.check_study_access(uuid, StudyPermissionType.READ, params)
+        return study_service.load_manager.get_load_matrix(study, area_id)
+
+    @bp.put(
+        "/studies/{uuid}/{area_id}/load/series",
+        tags=[APITag.study_data],
+        summary="Update load series data",
+    )
+    def update_load_series(
```

Comment:
If we want to support arrow input, I don't think we will be able to use a pydantic model (LoadDTO), because they expect JSON, which cannot contain binary data.

So: if we want the same URL to support at the same time JSON and arrow,  we will need  to separate it in 2 functions I think, for example discriminated by the "accept-content" header.
In any case, I think the function which will accept arrow input will need to have something else than a pydantic model as its argument (bytes or something else, I did not look into fastapi support for binary input).

Reply:
Eventually, we decided to only support Arrow format. I'll check to see what is best to handle binary inputs